.github/reusable-build/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Reusable action which builds struct2tensor
+
+inputs:
+  python-version:
+    description: 'Python version'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+  - name: Build the package for Python ${{ inputs.python-version }}
+    shell: bash
+    run: |
+      docker compose build --build-arg PYTHON_VERSION=${{ inputs.python-version }} manylinux2014
+      docker compose run -e TF_VERSION=RELEASED_TF_2 manylinux2014
+      bazel build -c opt struct2tensor:struct2tensor_kernels_and_ops
```

Comment:
While working on the testing workflow, I realised you'd need numpy as well, to get this working.

See the following: 
- https://github.com/google/struct2tensor/blob/4ce8b4672f4c37aba5cbfbb06b38868ab18946b3/WORKSPACE#L56
- https://github.com/google/struct2tensor/blob/4ce8b4672f4c37aba5cbfbb06b38868ab18946b3/third_party/python_configure.bzl#L425

Build failure on my fork: https://github.com/aktech/struct2tensor/actions/runs/11296563121/job/31424180115#step:3:3196

Reply:
Hmm, I don't understand. Isn't the container providing the build isolation? If it's failing to find `numpy`, I guess the container itself is not working correctly?

Reply:
I'm able to build locally, and I guess I made the wrong assumption that the isolation provided by the container would make it build remotely. Thanks so much for taking a look - let me try on a fork!

Reply:
...just realized I call `bazel build -c opt struct2tensor:struct2tensor_kernels_and_ops` here. So much for build isolation :roll_eyes: